dev.json

```diff
@@ -1,6 +1,6 @@
 {
   "service": {
-    "host": "localhost",
+    "host": "0.0.0.0",
```

Comment:
Nit; I guess it's not super important, but I'm not sure how fair is to set it like this. I've heard some complains from security folks for using `0.0.0.0` (_by default_) as it would listen for any incoming connection. Up to you!

Reply:
Is there any reason to make this change?

Reply:
Without this I wasn't able to connect to the renderer when running it inside Docker against a locally run Grafana.

Reply:
Now I'm actually wondering why is this file used by docker, as it should be only applied for development.

Reply:
Hmm I don't have this issue when I run the image renderer Docker image. How do you run it exactly?

Reply:
Not sure what happened, but I tried again and it works with localhost 🙈 I'll revert this change.

Reply:
Not sure what happened, but I tried again and it works with localhost 🙈 I'll revert this change.